src/common/API/storageAPI.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getQueueNumber = async (network: string): Promise<any> => {
+  const url = `api${network === "homestead" ? "" : `-${network}`}.tradetrust.io/storage/queue`;
+
+  try {
+    const response = await axios({
+      method: "get",
+      url: `https://${url}`,
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    return response.data;
+  } catch (e) {
```

Comment:
swallowing errors make things very hard to debug in the future when errors do happen

Reply:
understand this, so i have updated the files. but because there is a story to display the errors for publishing the doc, is it ok if i add the error handling for upload to storage into that story AC?

Reply:
no its fine to leave the feature for that AC for the other story, but when you catch(e) {} and don't show any error in console, sometime down the line we will be very confused when there's an error and nothing is being reported
so for e.g do a console.error(e) before proceeding with whatever

Reply:
ooohhhh! haha ok!